### ISSUE TITLE
unable to deal with fake-PNG (jpeg with .png extension)

### ISSUE THREAD

**Issue by fabiscafe** [pablo-s/passes#9]
So I tried a few old pkpasses. And I found that my cinemy includes movie pictures as thumbnails. The files are marked with a .png extension, but are in real jpeg.

The error is:
```
flatpak run me.sanchezrodriguez.passes
Traceback (most recent call last):
  File "/app/share/passes/passes/main.py", line 127, in _on_file_chosen
    pkpass = PassFactory.create(pkpass_file)
  File "/app/share/passes/passes/digital_pass_factory.py", line 41, in create
    digital_pass = this_class.create_pkpass(archive)
  File "/app/share/passes/passes/digital_pass_factory.py", line 65, in create_pkpass
    pixbuf = thisClass.create_pixbuf_from_filename(archive, file_name)
  File "/app/share/passes/passes/digital_pass_factory.py", line 102, in create_pixbuf_from_filename
    loader.write(image_data)
gi.repository.GLib.GError: gdk-pixbuf-error-quark: Fatal error reading PNG image file: Not a PNG file (0)
```

This is the content of a pkpass
```
❯ unzip pass.pkpass
Archive:  pass.pkpass
  inflating: thumbnail@3x.png        
  inflating: background@2x.png       
  inflating: icon.png                
  inflating: logo@2x.png             
  inflating: thumbnail@2x.png        
  inflating: background.png          
  inflating: icon@3x.png             
  inflating: logo.png                
  inflating: logo@3x.png             
  inflating: background@3x.png       
  inflating: icon@2x.png             
  inflating: thumbnail.png           
  inflating: pass.json               
  inflating: manifest.json           
  inflating: signature               
❯ file *.png
background@2x.png: PNG image data, 720 x 880, 8-bit colormap, non-interlaced
background@3x.png: PNG image data, 1080 x 1320, 8-bit colormap, non-interlaced
background.png:    PNG image data, 360 x 440, 1-bit colormap, non-interlaced
icon@2x.png:       PNG image data, 240 x 240, 8-bit/color RGB, non-interlaced
icon@3x.png:       PNG image data, 360 x 360, 8-bit/color RGB, non-interlaced
icon.png:          PNG image data, 120 x 120, 8-bit/color RGB, non-interlaced
logo@2x.png:       PNG image data, 400 x 200, 8-bit gray+alpha, non-interlaced
logo@3x.png:       PNG image data, 600 x 300, 8-bit gray+alpha, non-interlaced
logo.png:          PNG image data, 200 x 100, 8-bit gray+alpha, non-interlaced
thumbnail@2x.png:  JPEG image data, JFIF standard 1.01, resolution (DPI), density 96x96, segment length 16, progressive, precision 8, 320x451, components 3
thumbnail@3x.png:  JPEG image data, JFIF standard 1.01, resolution (DPI), density 96x96, segment length 16, progressive, precision 8, 480x677, components 3
thumbnail.png:     JPEG image data, JFIF standard 1.01, resolution (DPI), density 96x96, segment length 16, progressive, precision 8, 160x226, components 3
```

**Comment by pablo-s** [comment]
Thanks for reporting this bug @fabiscafe!
I have just released version 0.3 in Flathub with a fix for the same (or at least similar) problem.
It would be great if you could try to reproduce this bug in the latest version.

**Comment by fabiscafe** [comment]
With 0.3 I can not recreate this bug. Thank you so much :heart_decoration:
